src/server/scripts/Northrend/Ulduar/Ulduar/boss_assembly_of_iron.cpp

```diff
@@ -12,46 +12,46 @@
 enum AssemblySpells
 {
     // Any boss
-    SPELL_SUPERCHARGE               = 61920,
```

Comment:
Please restore the indentation for all enums to their original state in order to improve readability.

Reply:
The indentation is not fixed yet. And you could close your PRs #2055 and #2062 as they are contained already in this PR.